protocol/avdl/chat1/local.avdl

```diff
@@ -511,7 +511,7 @@ protocol local {
     MessageType messageType;
     gregor1.Time ctime;
     boolean isEphemeral;
-    boolean isEphemeralExpired;
+    union { null, string } explodedBy;
```

Comment:
Can we have both? This way JS side don't need to do the comparison.

Reply:
We'd have to have a derived type that computes the value before sending it to the frontend. To keep things simple I just let the frontent derive it, we do a similar calculation here https://github.com/keybase/client/pull/21876/files#diff-ff27bea5c579bc43b5b22d046db6acc5R1214